srcpkgs/font-vazir-code/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'font-vazir-code'
+pkgname=font-vazir-code
+version=1.1.2
+revision=1
+create_wrksrc=yes
+depends="font-util"
+short_desc="Persian (farsi) Monospaced Font for coding"
```

Comment:
I don't think it needs the "for coding" part.

Reply:
removed.